<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchFiltersMainWindow</class>
 <widget class="QMainWindow" name="BatchFiltersMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Piaf - Batch processor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../piaf.qrc">
    <normaloff>:/images/pixmaps/filter.png</normaloff>:/images/pixmaps/filter.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Batch files</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="filesTreeWidget">
         <property name="toolTip">
          <string>Files for batch processing</string>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="autoExpandDelay">
          <number>1</number>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>File</string>
          </property>
          <property name="icon">
           <iconset resource="../../piaf.qrc">
            <normaloff>:/images/pixmaps/movie.png</normaloff>:/images/pixmaps/movie.png</iconset>
          </property>
         </column>
         <column>
          <property name="text">
           <string>State</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="buttonsWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="toolTip">
             <string>Add new files</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset resource="../../piaf.qrc">
              <normaloff>:/images/22x22/list-add.png</normaloff>:/images/22x22/list-add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="toolTip">
             <string>Reset state to unprocessed</string>
            </property>
            <property name="text">
             <string>Reset</string>
            </property>
            <property name="icon">
             <iconset resource="../../piaf.qrc">
              <normaloff>:/images/22x22/edit-undo.png</normaloff>:/images/22x22/edit-undo.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delButton">
            <property name="toolTip">
             <string>Remove selected files</string>
            </property>
            <property name="text">
             <string>Del</string>
            </property>
            <property name="icon">
             <iconset resource="../../piaf.qrc">
              <normaloff>:/images/22x22/list-remove.png</normaloff>:/images/22x22/list-remove.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Plugins</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="loadButton">
            <property name="toolTip">
             <string>Load plugin sequence</string>
            </property>
            <property name="text">
             <string>Plugin sequence</string>
            </property>
            <property name="icon">
             <iconset resource="../../piaf.qrc">
              <normaloff>:/images/16x16/document-open.png</normaloff>:/images/16x16/document-open.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="sequenceLabel">
            <property name="text">
             <string>--</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="ImageWidget" name="imageLabel" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="controlGroupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Control</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="1" colspan="4">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="recordButton">
         <property name="toolTip">
          <string>Record output image or movies</string>
         </property>
         <property name="text">
          <string>Record</string>
         </property>
         <property name="icon">
          <iconset resource="../../piaf.qrc">
           <normaloff>:/images/22x22/media-record.png</normaloff>:/images/22x22/media-record.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="greyButton">
         <property name="toolTip">
          <string>if down, the images will be converted to grayscale befor eprocessing</string>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
         <property name="icon">
          <iconset resource="../../piaf.qrc">
           <normaloff>:/images/22x22/view-color.png</normaloff>
           <activeon>:/images/22x22/view-gray.png</activeon>:/images/22x22/view-color.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QPushButton" name="playPauseButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
         <property name="icon">
          <iconset resource="../../piaf.qrc">
           <normaloff>:/images/32x32/media-playback-start.png</normaloff>
           <activeoff>:/images/32x32/media-playback-pause.png</activeoff>:/images/32x32/media-playback-start.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="viewButton">
         <property name="toolTip">
          <string>Display the image output once per second</string>
         </property>
         <property name="text">
          <string>View</string>
         </property>
         <property name="icon">
          <iconset resource="../../piaf.qrc">
           <normaloff>:/images/16x16/layer-visible-on.png</normaloff>:/images/16x16/layer-visible-on.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QCheckBox" name="reloadPluginCheckBox">
         <property name="toolTip">
          <string>Reload plugins at every change of file</string>
         </property>
         <property name="statusTip">
          <string>Reload the plugins at every change of file. If unchecked, all the images will be processed by the same plugin processus (beware of image size changes).</string>
         </property>
         <property name="text">
          <string>1 plug/file</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Processing time histogram</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="histogramLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Status</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Files:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nbFilesLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Number of files in process list</string>
         </property>
         <property name="text">
          <string>--</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header location="global">imagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../piaf.qrc"/>
 </resources>
 <connections/>
</ui>
